rpc/src/transaction_status_service.rs

```diff
@@ -452,16 +452,15 @@ pub(crate) mod tests {
 
         exit.store(true, Ordering::Relaxed);
         transaction_status_service.join().unwrap();
-        let notifier = test_notifier.read().unwrap();
-        assert_eq!(notifier.notifications.len(), 1);
+        assert_eq!(test_notifier.notifications.len(), 1);
         let key = TestNotifierKey {
             slot,
             transaction_index,
             signature,
         };
-        assert!(notifier.notifications.contains_key(&key));
+        assert!(test_notifier.notifications.contains_key(&key));
 
-        let result = &*notifier.notifications.get(&key).unwrap();
+        let result = &*test_notifier.notifications.get(&key).unwrap();
```

Comment:
```suggestion
        let result = test_notifier.notifications.get(&key).unwrap();
```
I don't think this is necessary anymore.

Reply:
Done